index.html

```diff
@@ -0,0 +1,278 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>New-Year-party</title>
+    <link rel="stylesheet" href="styles.css" type="text/css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400&display=swap"
+      rel="stylesheet"
+    />
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+  </head>
+  <body>
+    <section class="main-container">
+      <section class="hello-container">
+        <div class="upper">
+          <a href="www.facebook.com" class="facebook"
```

Comment:
- [ ] Please make both, about and index pages responsive, currently they are not responsive, I recorded a short video to demonstrate the issue: 

https://user-images.githubusercontent.com/77038610/163391968-baa71c50-e097-4b4a-8a8e-1c9e6b508acf.mp4

More specifically: 

- [ ] Please make sure that in mobile size, there is no white space on the right side and content does not overflow. 
- [ ] Please arrange the content inside main program cards in a way so that they looked neat and nice, currently the text takes a lot of height. 
- [ ] In sponsors section make sure to fit the logo so that they fit the container. 
- [ ] Text inside the images in about section should be centred. 
- [ ] Finally, make sure that your design is very similar to the provided design (margins, paddings, colors)